prisma/schema.prisma

```diff
@@ -7,11 +7,62 @@ generator client {
   provider = "prisma-client-js"
 }
 
-model User {
-  username String @id
-  name     String
-  email    String @unique
+model Users {
```

Comment:
One more thing I missed before.  In Prisma, the suggestion is to use singular vs. plural forms for the names:

- User vs. Users
- Subdomain vs. Subdomains
- etc